ddocs/medic/_attachments/translations/messages-en.properties

```
@@ -648,6 +648,8 @@ export.messages.description = Download all messages that have ever been sent or
 export.people.description = Download all contacts registered in the system in JSON format.
 export.reports.description = Download a summary of all the reports that have ever been submitted in CSV format.
 extra_fields = Extra fields.
+external-app-launcher.button.launch = Launch app
+external-app-launcher.message.disable = Launching external apps is not available on this device.
```

Instead of mixing and matching between hyphens and full stops, can we stick to full stops? I think this may cause problems because `-` would have to be escaped in JS anyway.

```suggestion
external.app.launcher.button.launch = Launch app
external.app.launcher.message.disable = Launching external apps is not available on this device.
```

---

Actually, looking across the file, I think we replace the spaces with underscore, and then use dots for addressing elements, like:` health_center.field.children = Areas`

I'm going to update use underscore: 
```
external_app_launcher.button.launch = Launch app
external_app_launcher.message.disable = Launching external apps is not available on this device.
```